astro/src/content/docs/extend/events-and-webhooks/events/identity-verified.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Identity Verified
+description: Learn about the Identity Verified event.
+section: extend
+subcategory: events and webhooks
+tertcategory: events
+---
+import APIField from 'src/components/api/APIField.astro';
+import Aside from "src/components/Aside.astro";
+import Event from 'src/content/docs/extend/events-and-webhooks/events/_event.astro';
+import EventBody from 'src/content/docs/extend/events-and-webhooks/events/_event-body.astro';
+import InlineField from 'src/components/InlineField.astro';
+
+export const eventType = 'identity.verified';
+
+<Event description="This event is generated when a user verifies an identity, which could be an email address or phone number."
+       eventType={eventType}
+       name="Identity Verified"
+       scope="tenant"
+       transactional="true"
+       version="1.99.9"/>
+
+<Aside>
+  The [User Email Verified](/docs/extend/events-and-webhooks/events/user-email-verified) event is also created when email identities are verified and will continue to function for compatibility reasons. This event is more generic and will be used for all identity verifications, including phone numbers.
```

Comment:
we're not formally deprecating email verified (I don't know that we need to) so tried to explain this